web-client/src/presenter/state.ts

```diff
@@ -575,6 +575,7 @@ export const baseState = {
     zipInProgress?: boolean;
     totalFiles?: number;
     fileCount?: number;
+    title: string;
```

Comment:
Make optional, we just introduced this property. So only this feature is setting it